packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -347,18 +347,22 @@ export const makeAnchorAsset = async (
     }),
   );
 
-  const { creatorFacet: mint, publicFacet: issuer } = /**
-   * @type {{
+  /**
+   * @typedef {{
    *   creatorFacet: ERef<Mint<'nat'>>;
    *   publicFacet: ERef<Issuer<'nat'>>;
-   * }}
-   */ (
-    await E(startUpgradable)({
-      installation: mintHolder,
-      label: keyword,
-      terms,
-    })
-  );
+   * }} StartUpgradableResponse
```

Comment:
This is specific to the PSM
```suggestion
   * }} PsmKit
```